drupal/.lando/update.sh

```diff
@@ -18,7 +18,7 @@ drush fra -y
 
 # Enable development modules
 drush en -y stage_file_proxy update devel
-drush vset stage_file_proxy_origin 'https://www.pori.fi'
+drush vset stage_file_proxy_origin 'https://pori.prod.wunder.io'
```

Comment:
Site is put under basic auth. Might be this will not work.

Reply:
Correct. Ho can we bypass basic auth at least for our own VPN?

Reply:
Done.